go/libkb/interfaces.go

```diff
@@ -606,10 +606,31 @@ type TeamLoader interface {
 	ClearMem()
 }
 
+type ImplicitTeamConflictInfoCacher interface {
+	Get(context.Context, bool, keybase1.TeamID) *keybase1.ImplicitTeamConflictInfo
+	Put(context.Context, bool, keybase1.TeamID, keybase1.ImplicitTeamConflictInfo) error
+}
+
 type KVStoreContext interface {
 	GetKVStore() KVStorer
 }
 
+type LRUContext interface {
+	VLogContext
+	KVStoreContext
+	ClockContext
+}
+
+type LRUKeyer interface {
+	MemKey() string
+	DbKey() DbKey
```

Comment:
For a bit of convenience `MemKey` could be omitted from the interface and always use `DbKey().Key`.